/kaggle/input/fer-2013/train
/kaggle/input/fer-2013/test
48
Traceback (most recent call last):
  File "/kaggle/working/channel_and_spatial_attentions_with_cnn/resnet.py", line 193, in <module>
    main()
  File "/kaggle/working/channel_and_spatial_attentions_with_cnn/resnet.py", line 94, in main
    train_generator = training_datagen.flow_from_directory(TRAINING_DIR, target_size=(img_size, img_size),
  File "/usr/local/lib/python3.10/dist-packages/keras/src/legacy/preprocessing/image.py", line 1138, in flow_from_directory
    return DirectoryIterator(
  File "/usr/local/lib/python3.10/dist-packages/keras/src/legacy/preprocessing/image.py", line 453, in __init__
    for subdir in sorted(os.listdir(directory)):
FileNotFoundError: [Errno 2] No such file or directory: '/kaggle/input/fer-2013/train'
